src/test/java/org/folio/bulkops/TestEntity.java

```diff
@@ -85,11 +84,11 @@ public BulkOperationsEntity entity() {
         .numberOfMissingPieces("1")
         .missingPieces("missing")
         .missingPiecesDate("01-01-2000")
-        .itemDamagedStatusId("damaged-status-id")
+        .itemDamagedStatus("damaged-status-id")
         .itemDamagedStatusDate("01-2001")
         .administrativeNotes(List.of("administrative-note-1", "administrative-note-2"))
         .notes(List.of(ItemNote.builder().build()))
-        .circulationNotes(List.of(CirculationNote.builder().build()))
+//        .circulationNotes(List.of(CirculationNote.builder().build()))
```

Comment:
@AntonAntonich please uncomment the code logic

Reply:
done